In [35]:
#A class attribute that provides access to the TabularDatasetFactory methods for creating new TabularDataset objects. 
#Usage: Dataset.Tabular.from_delimited_files().
from azureml.core import Workspace, Dataset

subscription_id = 'bcfe0c62-8ebe-4df0-a46d-1efcf8739a5b' #check the launch studio there will get this id
resource_group = 'VChamp-Team3' # resource group name
workspace_name = 'vchamp-team3' # worksapce name


# storage account : Algorithmia, Resource group: VChamp-Team3 and workspace: vchamp-team3.
#Constructor
workspace = Workspace(subscription_id, resource_group, workspace_name)

In [36]:
#['data_team3_synthetic_train']
datastore = workspace.datastores['data_team3_synthetic_quality_check'] 

In [37]:
#from_delimited_files (Create a TabularDataset to represent tabular data in delimited files (e.g. CSV and TSV).

dataset = Dataset.Tabular.from_delimited_files(path = [(datastore, 'immunization_qual.csv')])

# preview the first 3 rows of the dataset
# dataset.to_pandas_dataframe()

In [38]:
#Converting the dataset into data frame(default as dataset in Azure, thus we must convert the needed formate)
immunizations_train = dataset.to_pandas_dataframe()

In [39]:
immunizations_train

Column1  Internalpatientid  Age at immunization   Immunization date  \
0          194             100229            68.021827 2000-09-25 23:27:52   
1          195             100229            68.021827 2000-09-25 23:27:52   
2          253             100314            64.986914 2011-07-04 17:12:36   
3          302             100399            77.128884 2001-08-23 14:34:16   
4          303             100399            78.234635 2002-10-01 18:13:15   
...        ...                ...                  ...                 ...   
13580  2346260              99143            90.632019 2012-01-15 00:04:27   
13581  2346516               9947            67.200942 2010-10-16 22:39:38   
13582  2346517               9947            67.200942 2010-10-16 22:39:38   
13583  2346878              99941            76.400998 2018-12-19 13:33:08   
13584  2346881              99944            80.397951 2012-12-01 10:49:45   

                                Immunization Dose quantity Dose unit  \
0      PNEUMOCOCCAL, UNSPECIFIED FORMULATION          None      None   
1         INFLUENZA, UNSPECIFIED FORMULATION          None      None   
2         INFLUENZA, UNSPECIFIED FORMULATION          None      None   
3         INFLUENZA, UNSPECIFIED FORMULATION          None      None   
4          TD(ADULT) UNSPECIFIED FORMULATION          None      None   
...                                      ...           ...       ...   
13580     INFLUENZA, UNSPECIFIED FORMULATION          None      None   
13581     INFLUENZA, UNSPECIFIED FORMULATION          None      None   
13582                 INFLUENZA (HISTORICAL)          None      None   
13583        INFLUENZA, SEASONAL, INJECTABLE          None      None   
13584  PNEUMOCOCCAL, UNSPECIFIED FORMULATION          None      None   

      Administered elsewhere  Cvx code              Series doses  \
0                       None     109.0  Not specified (no value)   
1                       None      88.0  Not specified (no value)   
2                       None      88.0  Not specified (no value)   
3                       None      88.0  Not specified (no value)   
4                       None     139.0  Not specified (no value)   
...                      ...       ...                       ...   
13580                   None      88.0  Not specified (no value)   
13581                   None      88.0  Not specified (no value)   
13582                   None      88.0  Not specified (no value)   
13583                   None     141.0  Not specified (no value)   
13584                   None     109.0  Not specified (no value)   

               State  
0            Arizona  
1            Arizona  
2           Missouri  
3          Minnesota  
4          Minnesota  
...              ...  
13580        Florida  
13581           Ohio  
13582           Ohio  
13583          Maine  
13584  Massachusetts  

[13585 rows x 11 columns]

In [40]:
import numpy as np                # Multi-Dimensional array object
import pandas as pd               # Data Manipulation
import matplotlib.pyplot as plt   # Data Visualization
import seaborn as sns             # Data Visualization 

# Allows the use of display() for DataFrames
from IPython.display import display 

# Handle warning messages that may occur during code execution
import warnings
warnings.filterwarnings('ignore')

# To increase memory allocation
from notebook.services.config import ConfigManager
cm = ConfigManager().update('notebook', {'max_buffer_size': 7294967296})

# Set the maximum number of rows to be displayed to 1000
pd.set_option('display.max_rows', 1163)

# Elimination of the unnamed columns

In [41]:
# Remove unwanted columns 
immunizations_train.drop('Column1',axis=1,inplace=True)

In [42]:
immunizations_train = immunizations_train

In [43]:
immunizations_train

Internalpatientid  Age at immunization   Immunization date  \
0                 100229            68.021827 2000-09-25 23:27:52   
1                 100229            68.021827 2000-09-25 23:27:52   
2                 100314            64.986914 2011-07-04 17:12:36   
3                 100399            77.128884 2001-08-23 14:34:16   
4                 100399            78.234635 2002-10-01 18:13:15   
...                  ...                  ...                 ...   
13580              99143            90.632019 2012-01-15 00:04:27   
13581               9947            67.200942 2010-10-16 22:39:38   
13582               9947            67.200942 2010-10-16 22:39:38   
13583              99941            76.400998 2018-12-19 13:33:08   
13584              99944            80.397951 2012-12-01 10:49:45   

                                Immunization Dose quantity Dose unit  \
0      PNEUMOCOCCAL, UNSPECIFIED FORMULATION          None      None   
1         INFLUENZA, UNSPECIFIED FORMULATION          None      None   
2         INFLUENZA, UNSPECIFIED FORMULATION          None      None   
3         INFLUENZA, UNSPECIFIED FORMULATION          None      None   
4          TD(ADULT) UNSPECIFIED FORMULATION          None      None   
...                                      ...           ...       ...   
13580     INFLUENZA, UNSPECIFIED FORMULATION          None      None   
13581     INFLUENZA, UNSPECIFIED FORMULATION          None      None   
13582                 INFLUENZA (HISTORICAL)          None      None   
13583        INFLUENZA, SEASONAL, INJECTABLE          None      None   
13584  PNEUMOCOCCAL, UNSPECIFIED FORMULATION          None      None   

      Administered elsewhere  Cvx code              Series doses  \
0                       None     109.0  Not specified (no value)   
1                       None      88.0  Not specified (no value)   
2                       None      88.0  Not specified (no value)   
3                       None      88.0  Not specified (no value)   
4                       None     139.0  Not specified (no value)   
...                      ...       ...                       ...   
13580                   None      88.0  Not specified (no value)   
13581                   None      88.0  Not specified (no value)   
13582                   None      88.0  Not specified (no value)   
13583                   None     141.0  Not specified (no value)   
13584                   None     109.0  Not specified (no value)   

               State  
0            Arizona  
1            Arizona  
2           Missouri  
3          Minnesota  
4          Minnesota  
...              ...  
13580        Florida  
13581           Ohio  
13582           Ohio  
13583          Maine  
13584  Massachusetts  

[13585 rows x 10 columns]

In [46]:
immunizations_train["Internalpatientid"].nunique()

939

# sorting out patients id and age column

In [47]:
# Sort the dataset based on the patientsid and 'age' column in ascending order
immunizations_train.sort_values(["Internalpatientid","Age at immunization"],inplace=True)

In [48]:
immunizations_train

Internalpatientid  Age at immunization   Immunization date  \
12616                 67            48.284719 2010-08-10 01:33:24   
12617                 67            48.284719 2010-08-10 01:33:24   
9187                  67            48.305034 2010-08-17 11:45:26   
9051                  67            49.301203 2011-08-16 14:09:21   
8450                  67            49.940158 2012-04-06 03:03:52   
...                  ...                  ...                 ...   
5347              168899            89.554915 2012-05-17 07:42:45   
5342              168899            90.568326 2013-05-22 17:21:10   
11202             168899            92.401948 2015-03-23 21:53:17   
10848             168899            93.483974 2016-04-22 09:24:59   
10849             168899            93.483974 2016-04-22 09:24:59   

                                            Immunization Dose quantity  \
12616                             FLU,3 YRS (HISTORICAL)          None   
12617   TDAP (Tetanus,Diphtheria,Pertussis) (HISTORICAL)          None   
9187               PNEUMOCOCCAL, UNSPECIFIED FORMULATION          None   
9051                              FLU,3 YRS (HISTORICAL)          None   
8450                              FLU,3 YRS (HISTORICAL)          None   
...                                                  ...           ...   
5347                              FLU,3 YRS (HISTORICAL)          None   
5342                              FLU,3 YRS (HISTORICAL)          None   
11202                             FLU,3 YRS (HISTORICAL)          None   
10848                      PNEUMOCOCCAL CONJUGATE PCV 13          None   
10849  INFLUENZA, SEASONAL, INJECTABLE, PRESERVATIVE ...          None   

      Dose unit Administered elsewhere  Cvx code              Series doses  \
12616      None                   None      88.0  Not specified (no value)   
12617      None                   None     115.0  Not specified (no value)   
9187       None                   None     109.0  Not specified (no value)   
9051       None                   None      88.0  Not specified (no value)   
8450       None                   None      88.0  Not specified (no value)   
...         ...                    ...       ...                       ...   
5347       None                   None      88.0  Not specified (no value)   
5342       None                   None      88.0  Not specified (no value)   
11202      None        Non-VA Location      88.0  Not specified (no value)   
10848      None                   None     133.0  Not specified (no value)   
10849      None                   None     140.0  Not specified (no value)   

              State  
12616    California  
12617    California  
9187     California  
9051     California  
8450     California  
...             ...  
5347   Pennsylvania  
5342   Pennsylvania  
11202  Pennsylvania  
10848  Pennsylvania  
10849  Pennsylvania  

[13585 rows x 10 columns]

# Checking the missing value

In [49]:
print("Training set missing values:\n", immunizations_train.isna().sum())

Training set missing values:
 Internalpatientid             0
Age at immunization           0
Immunization date             0
Immunization                  0
Dose quantity             13558
Dose unit                 13558
Administered elsewhere    11640
Cvx code                    121
Series doses                  0
State                         0
dtype: int64


* **There are many missing values in the column "Dose quantity"(1836499),"Dose unit"(1836502),"Administered elsewhere"(1585089) "Cvx code"(121), and "Immunization" (3) And other varible have no missing values in this dataset.**

**Note:Here potiential attributes is Immunization, so it will have three mising value we can remove.**

# Removing the missing value

In [50]:
# Remove rows missing values in the "Immunization" column
immunizations_train.dropna(subset=["Immunization"],inplace=True)

In [51]:
immunizations_train = immunizations_train

In [52]:
immunizations_train

Internalpatientid  Age at immunization   Immunization date  \
12616                 67            48.284719 2010-08-10 01:33:24   
12617                 67            48.284719 2010-08-10 01:33:24   
9187                  67            48.305034 2010-08-17 11:45:26   
9051                  67            49.301203 2011-08-16 14:09:21   
8450                  67            49.940158 2012-04-06 03:03:52   
...                  ...                  ...                 ...   
5347              168899            89.554915 2012-05-17 07:42:45   
5342              168899            90.568326 2013-05-22 17:21:10   
11202             168899            92.401948 2015-03-23 21:53:17   
10848             168899            93.483974 2016-04-22 09:24:59   
10849             168899            93.483974 2016-04-22 09:24:59   

                                            Immunization Dose quantity  \
12616                             FLU,3 YRS (HISTORICAL)          None   
12617   TDAP (Tetanus,Diphtheria,Pertussis) (HISTORICAL)          None   
9187               PNEUMOCOCCAL, UNSPECIFIED FORMULATION          None   
9051                              FLU,3 YRS (HISTORICAL)          None   
8450                              FLU,3 YRS (HISTORICAL)          None   
...                                                  ...           ...   
5347                              FLU,3 YRS (HISTORICAL)          None   
5342                              FLU,3 YRS (HISTORICAL)          None   
11202                             FLU,3 YRS (HISTORICAL)          None   
10848                      PNEUMOCOCCAL CONJUGATE PCV 13          None   
10849  INFLUENZA, SEASONAL, INJECTABLE, PRESERVATIVE ...          None   

      Dose unit Administered elsewhere  Cvx code              Series doses  \
12616      None                   None      88.0  Not specified (no value)   
12617      None                   None     115.0  Not specified (no value)   
9187       None                   None     109.0  Not specified (no value)   
9051       None                   None      88.0  Not specified (no value)   
8450       None                   None      88.0  Not specified (no value)   
...         ...                    ...       ...                       ...   
5347       None                   None      88.0  Not specified (no value)   
5342       None                   None      88.0  Not specified (no value)   
11202      None        Non-VA Location      88.0  Not specified (no value)   
10848      None                   None     133.0  Not specified (no value)   
10849      None                   None     140.0  Not specified (no value)   

              State  
12616    California  
12617    California  
9187     California  
9051     California  
8450     California  
...             ...  
5347   Pennsylvania  
5342   Pennsylvania  
11202  Pennsylvania  
10848  Pennsylvania  
10849  Pennsylvania  

[13585 rows x 10 columns]

In [53]:
# Checking the missing value after removing 
print("Training set missing values:\n", immunizations_train.isna().sum())

Training set missing values:
 Internalpatientid             0
Age at immunization           0
Immunization date             0
Immunization                  0
Dose quantity             13558
Dose unit                 13558
Administered elsewhere    11640
Cvx code                    121
Series doses                  0
State                         0
dtype: int64


# Taking the one digit in age coulmn

In [54]:
# Format 'Age at measurement' values in the format 00.00
immunizations_train["Age at immunization"] = immunizations_train["Age at immunization"].map("{:.2f}".format)

In [55]:
immunizations_train

Internalpatientid Age at immunization   Immunization date  \
12616                 67               48.28 2010-08-10 01:33:24   
12617                 67               48.28 2010-08-10 01:33:24   
9187                  67               48.31 2010-08-17 11:45:26   
9051                  67               49.30 2011-08-16 14:09:21   
8450                  67               49.94 2012-04-06 03:03:52   
...                  ...                 ...                 ...   
5347              168899               89.55 2012-05-17 07:42:45   
5342              168899               90.57 2013-05-22 17:21:10   
11202             168899               92.40 2015-03-23 21:53:17   
10848             168899               93.48 2016-04-22 09:24:59   
10849             168899               93.48 2016-04-22 09:24:59   

                                            Immunization Dose quantity  \
12616                             FLU,3 YRS (HISTORICAL)          None   
12617   TDAP (Tetanus,Diphtheria,Pertussis) (HISTORICAL)          None   
9187               PNEUMOCOCCAL, UNSPECIFIED FORMULATION          None   
9051                              FLU,3 YRS (HISTORICAL)          None   
8450                              FLU,3 YRS (HISTORICAL)          None   
...                                                  ...           ...   
5347                              FLU,3 YRS (HISTORICAL)          None   
5342                              FLU,3 YRS (HISTORICAL)          None   
11202                             FLU,3 YRS (HISTORICAL)          None   
10848                      PNEUMOCOCCAL CONJUGATE PCV 13          None   
10849  INFLUENZA, SEASONAL, INJECTABLE, PRESERVATIVE ...          None   

      Dose unit Administered elsewhere  Cvx code              Series doses  \
12616      None                   None      88.0  Not specified (no value)   
12617      None                   None     115.0  Not specified (no value)   
9187       None                   None     109.0  Not specified (no value)   
9051       None                   None      88.0  Not specified (no value)   
8450       None                   None      88.0  Not specified (no value)   
...         ...                    ...       ...                       ...   
5347       None                   None      88.0  Not specified (no value)   
5342       None                   None      88.0  Not specified (no value)   
11202      None        Non-VA Location      88.0  Not specified (no value)   
10848      None                   None     133.0  Not specified (no value)   
10849      None                   None     140.0  Not specified (no value)   

              State  
12616    California  
12617    California  
9187     California  
9051     California  
8450     California  
...             ...  
5347   Pennsylvania  
5342   Pennsylvania  
11202  Pennsylvania  
10848  Pennsylvania  
10849  Pennsylvania  

[13585 rows x 10 columns]

In [56]:
# Convert 'Age at measurement' column from object to float
immunizations_train["Age at immunization"] = immunizations_train["Age at immunization"].astype(float)

In [57]:
immunizations_train

Internalpatientid  Age at immunization   Immunization date  \
12616                 67                48.28 2010-08-10 01:33:24   
12617                 67                48.28 2010-08-10 01:33:24   
9187                  67                48.31 2010-08-17 11:45:26   
9051                  67                49.30 2011-08-16 14:09:21   
8450                  67                49.94 2012-04-06 03:03:52   
...                  ...                  ...                 ...   
5347              168899                89.55 2012-05-17 07:42:45   
5342              168899                90.57 2013-05-22 17:21:10   
11202             168899                92.40 2015-03-23 21:53:17   
10848             168899                93.48 2016-04-22 09:24:59   
10849             168899                93.48 2016-04-22 09:24:59   

                                            Immunization Dose quantity  \
12616                             FLU,3 YRS (HISTORICAL)          None   
12617   TDAP (Tetanus,Diphtheria,Pertussis) (HISTORICAL)          None   
9187               PNEUMOCOCCAL, UNSPECIFIED FORMULATION          None   
9051                              FLU,3 YRS (HISTORICAL)          None   
8450                              FLU,3 YRS (HISTORICAL)          None   
...                                                  ...           ...   
5347                              FLU,3 YRS (HISTORICAL)          None   
5342                              FLU,3 YRS (HISTORICAL)          None   
11202                             FLU,3 YRS (HISTORICAL)          None   
10848                      PNEUMOCOCCAL CONJUGATE PCV 13          None   
10849  INFLUENZA, SEASONAL, INJECTABLE, PRESERVATIVE ...          None   

      Dose unit Administered elsewhere  Cvx code              Series doses  \
12616      None                   None      88.0  Not specified (no value)   
12617      None                   None     115.0  Not specified (no value)   
9187       None                   None     109.0  Not specified (no value)   
9051       None                   None      88.0  Not specified (no value)   
8450       None                   None      88.0  Not specified (no value)   
...         ...                    ...       ...                       ...   
5347       None                   None      88.0  Not specified (no value)   
5342       None                   None      88.0  Not specified (no value)   
11202      None        Non-VA Location      88.0  Not specified (no value)   
10848      None                   None     133.0  Not specified (no value)   
10849      None                   None     140.0  Not specified (no value)   

              State  
12616    California  
12617    California  
9187     California  
9051     California  
8450     California  
...             ...  
5347   Pennsylvania  
5342   Pennsylvania  
11202  Pennsylvania  
10848  Pennsylvania  
10849  Pennsylvania  

[13585 rows x 10 columns]

# Max age

In [59]:
# Find the maximum age for each internal patient id
max_ages = immunizations_train.groupby('Internalpatientid')['Age at immunization'].max().reset_index()

# Merge with the original dataframe to get the rows with the highest age
immunizations_train = pd.merge(immunizations_train, max_ages, on =['Internalpatientid','Age at immunization'], how = 'inner')

# Rounding off the Age 

In [60]:
immunizations_train['Age at immunization'] = immunizations_train['Age at immunization'].apply(lambda x: round(x,))

In [61]:
len(immunizations_train["Internalpatientid"].value_counts())

939

# Adding  'Age at immunization' & 'Immunization' columns with '_'

In [62]:
#In this code, we create a new column called 'Lab test description_Result numeric' by concatenating the values of the 'lab test description' column, underscore ('_'), and the 'lab test results' column
#converted to a string using the `astype(str)` function(because we cant't add differnt dtype)
immunizations_train['immunization'] =  immunizations_train['Age at immunization'].astype(str) + '_' +  immunizations_train['Immunization']

# Dropping all unnecessary columns

In [63]:
# Remove columns 
immunizations_train.drop(['Age at immunization','Immunization date','Immunization','Dose quantity','Dose unit','Administered elsewhere','Cvx code','Series doses','State'], axis=1,inplace=True)

In [64]:
immunizations_train = immunizations_train

In [65]:
# Group the DataFrame by 'Internalpatientid' and concatenate the icd10 values and
#The purpose of `x.dropna()` is to remove any missing values from the Series before applying a subsequent operation,
# such as concatenation using the `join` function. By dropping the missing values,
#we ensure that only non-null values are included in the resulting concatenated string.
df_grouped = immunizations_train.groupby('Internalpatientid').agg(lambda x: ','.join(x.dropna()))
df_grouped 

# Reset the index of the grouped DataFrame
df_grouped_imm = df_grouped.reset_index()
df_grouped_imm

Internalpatientid                                       immunization
0                   67  58_COVID-19 (PFIZER), MRNA, LNP-S, PF, 30 MCG/...
1                  200     87_INFLUENZA VACCINE, QUADRIVALENT, ADJUVANTED
2                  291  82_FLU,3 YRS (HISTORICAL),82_INFLUENZA, UNSPEC...
3                  330  75_COVID-19 (PFIZER), MRNA, LNP-S, PF, 30 MCG/...
4                  351                                            82_TDAP
5                  714              67_INFLUENZA, UNSPECIFIED FORMULATION
6                  785  78_INFLUENZA, UNSPECIFIED FORMULATION,78_FLU,3...
7                  960               60_TD(ADULT) UNSPECIFIED FORMULATION
8                 1429  81_PNEUMOCOCCAL, UNSPECIFIED FORMULATION,81_IN...
9                 1451  86_COVID-19 (MODERNA), MRNA, LNP-S, PF, 100 MC...
10                1477              84_INFLUENZA, UNSPECIFIED FORMULATION
11                1568  77_COVID-19 (MODERNA), MRNA, LNP-S, PF, 100 MC...
12                1805  83_INFLUENZA, INJECTABLE, QUADRIVALENT, PRESER...
13                2282              86_INFLUENZA, UNSPECIFIED FORMULATION
14                2500                          62_FLU,3 YRS (HISTORICAL)
15                2674  73_COVID-19 (MODERNA), MRNA, LNP-S, PF, 100 MC...
16                2993  85_FLU,3 YRS (HISTORICAL),85_INFLUENZA, UNSPEC...
17                3020              80_INFLUENZA, UNSPECIFIED FORMULATION
18                3450  70_COVID-19 (MODERNA), MRNA, LNP-S, PF, 100 MC...
19                3547  78_INFLUENZA, UNSPECIFIED FORMULATION,78_INFLU...
20                3606  64_PNEUMOCOCCAL POLYSACCHARIDE PPV23,64_INFLUE...
21                4128  80_PNEUMOCOCCAL, UNSPECIFIED FORMULATION,80_FL...
22                4177           85_PNEUMOCOCCAL, UNSPECIFIED FORMULATION
23                4203                              69_ZOSTER RECOMBINANT
24                4246                   74_PNEUMOCOCCAL CONJUGATE PCV 13
25                4453  74_FLU,3 YRS (HISTORICAL),74_INFLUENZA, UNSPEC...
26                4641                              84_ZOSTER RECOMBINANT
27                4845                67_INFLUENZA, TRIVALENT, ADJUVANTED
28                5653                              73_ZOSTER RECOMBINANT
29                5872              48_INFLUENZA, UNSPECIFIED FORMULATION
30                5920                 82_INFLUENZA, SEASONAL, INJECTABLE
31                6318              85_INFLUENZA, UNSPECIFIED FORMULATION
32                6370  104_COVID-19 (MODERNA), MRNA, LNP-S, PF, 100 M...
33                6400              77_INFLUENZA, UNSPECIFIED FORMULATION
34                6421  71_COVID-19 (PFIZER), MRNA, LNP-S, PF, 30 MCG/...
35                6584  82_COVID-19 (MODERNA), MRNA, LNP-S, PF, 100 MC...
36                6830              85_INFLUENZA, UNSPECIFIED FORMULATION
37                6859                          78_FLU,3 YRS (HISTORICAL)
38                7185              87_INFLUENZA, UNSPECIFIED FORMULATION
39                7232              75_INFLUENZA, UNSPECIFIED FORMULATION
40                7320     76_INFLUENZA VACCINE, QUADRIVALENT, ADJUVANTED
41                7572           83_TDAP,83_INFLUENZA, HIGH DOSE SEASONAL
42                7792  52_TD(ADULT) UNSPECIFIED FORMULATION,52_PNEUMO...
43                7925                          84_FLU,3 YRS (HISTORICAL)
44                8281  74_INFLUENZA, INJECTABLE, QUADRIVALENT, PRESER...
45                8286  82_COVID-19 (MODERNA), MRNA, LNP-S, PF, 100 MC...
46                8320                   79_INFLUENZA, HIGH DOSE SEASONAL
47                8418                          75_INFLUENZA (HISTORICAL)
48                8596                          96_INFLUENZA (HISTORICAL)
49                9391              86_INFLUENZA, UNSPECIFIED FORMULATION
50                9535                    75_INFLUENZA A & B (HISTORICAL)
51                9929               74_TD(ADULT) UNSPECIFIED FORMULATION
52                9947                                    76_TDAP,76_DTAP
53       

In [32]:
# Checking row values if it is correctly printed like A, B, C...
column_value = df_grouped_imm['immunization'].values[5]
print(column_value)

69_COVID-19 (PFIZER), MRNA, LNP-S, PF, 30 MCG/0.3 ML DOSE, TRIS-SUCROSE (AGES 12+ YEARS)


In [66]:
df_grouped_imm.to_csv('/mnt/batch/tasks/shared/LS_root/mounts/clusters/team-3-susai-gpu2/code/Users/900379/Output_files_quality/df_immunizations_quality.csv')